example/src/main/scala/xs4s/example/FindMostPopularWikipediaKeywordsZIOApp.scala

```diff
@@ -0,0 +1,56 @@
+package xs4s.example
+
+import zio._
+import zio.console._
+import zio.stream._
+import zio.blocking.Blocking
+import java.io.IOException
+import java.io.InputStream
+import java.net.URL
+import java.net.URLConnection
+import java.util.zip.GZIPInputStream
+import java.util.zip.InflaterInputStream
+import java.util.zip.Inflater
+import scala.collection.JavaConverters._
+import xs4s.ziocompat._
+import xs4s.syntax.zio._
+
+object FindMostPopularWikipediaKeywordsZIOApp extends zio.App {
+
+  def run(args: List[String]): URIO[ZEnv, ExitCode] = {
+
+    wikipediaXmlBytes
+      .via(byteStreamToXmlEventStream()(_))
+      .via(s => anchorExtractor.toZIOPipeThrowError(s) )
+      .via(stream => // by default, get a sub-set of the stream
+        if (args.contains("full")) stream else stream.take(500)
+      )
+      .map(_.text)
+      .via(countTopItemsZIO(_))
+      .foreach{ a => ZStream.fromIterable(a).foreach{ case (elem: String, count: Int) => putStrLn((count, elem).toString) } }
```

Comment:
is it possible to flatten the nesting here, ie 
.flatMap(counts => ZStream.fromIterable(counts))
.flatMap{case (elem, count) => putStrLn(...)}

I assume putStrLn returns an effect?

Reply:
```
      .flatMap(counts => ZStream.fromIterable(counts))
      .foreach{ case (elem: String, count: Int) => putStrLn((count, elem).toString) }
```

will update